src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -341,8 +342,19 @@ def _find_or_create_imported_symbol(location, name, scope_limit=None,
         return first_symbol_table.new_symbol(
                 name, interface=UnresolvedInterface(), **kargs)
 
+    # Are there any interfaces that might be hiding the symbol declaration?
+    symbol_table = location.scope.symbol_table
+    try:
+        _ = symbol_table.lookup("_psyclone_internal_interface")
+        # There is an unknown interface so add this symbol.
+        return location.scope.symbol_table.new_symbol(
```

Comment:
This can just be `symbol_table.new_symbol`, since you already made that variable

Reply:
Oh yes, thanks @LonelyCat124